core/bcast/bcast.go

```diff
@@ -144,14 +145,35 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty, pubkey core.
 		// Randao is an internal duty, not broadcasted to beacon chain
 		return nil
 	case core.DutyPrepareAggregator:
-		subscription, ok := aggData.(core.SignedBeaconCommitteeSubscription)
+		sub, ok := aggData.(core.SignedBeaconCommitteeSubscription)
 		if !ok {
-			return errors.New("invalid beacon committee subscription")
+			return errors.New("invalid beacon committee sub")
 		}
 
-		_, err = b.eth2Cl.SubmitBeaconCommitteeSubscriptionsV2(ctx, []*eth2exp.BeaconCommitteeSubscription{&subscription.BeaconCommitteeSubscription})
-		if err == nil {
-			log.Info(ctx, "Beacon committee subscription successfully submitted to beacon node", nil)
+		_, err = b.eth2Cl.SubmitBeaconCommitteeSubscriptionsV2(ctx, []*eth2exp.BeaconCommitteeSubscription{&sub.BeaconCommitteeSubscription})
+		if err != nil && strings.Contains(err.Error(), "404") {
```

Comment:
I think we can just try on any error, little simpler

Reply:
can also do:

```
if err == nil {
  return nil
}
log.Debug(ctx, "v2 failed ..."
// do v1
```

Reply:
Done!